src/libstore/ssh-store.cc

```diff
@@ -25,6 +25,8 @@ class SSHStore : public virtual SSHStoreConfig, public virtual RemoteStore
 {
 public:
 
+    const Setting<int> logFD{(StoreConfig*) this, -1, "log-fd", "file descriptor to which SSH's stderr is connected"};
+
```

Comment:
This, like all such settings, should go in `*StoreConfig` not `*Store`. Thus in this case it should go above in `SSHStoreConfig`,

Reply:
Oh I see the other one doesn't have this, in that case you should include the same comment.

Reply:
Seems to be fine and more correct here, so I fixed it, thanks!